frontend/.eslintrc

```diff
@@ -12,7 +12,7 @@
     "jsx-a11y/label-has-for": 0,
     "jsx-a11y/no-autofocus": 0,
     "jsx-a11y/no-static-element-interactions": 0,
-    "lodash/prefer-lodash-method": 1,
+    "lodash/prefer-lodash-method": [1, {"ignoreObjects": ["React\\.Children"]}],
```

Comment:
`React.Children` isn't really an array but rather an interface to a more complicated data structure. So when the linter complains about replacing `React.Children.map` and its ilk with lodash, it's mistaken.